SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+The FreeCAD project is a FOSS (Free and Open-Source Software) project that has a community of thousands of users and 
+hundreds of developers worldwide. We encourage responsible reporting of security vulnerabilities that may affect users
+of this software, and will endeavor to address these vulnerabilities when they are discovered.
+
+## Bounties
+
+We do not pay bounties to identify bugs, and we feel that it sets a dangerous precedent.  It encourages people to
+withhold information that is valuable to all in order to receive a personal financial benefit.  This runs against the
```

Comment:
is it really a good idea to so fundamentally oppose people getting paid for work? I understand stating we don't have bounties to offer, but do we really need this text here?
Does paying people for their work really encourage withholding information? Where has this been observed?
Aren't bounties payed _after_ disclosing such information?

Reply:
If you're expecting that there may be bounties in the future, the statement can sound downright hypocritical. We need to be clear about what we're opposing here.

Reply:
The genesis of that text is that a few weeks ago we received a "vulnerability report" (the results of an automated scan that identified ways we could improve our email hosting) along with a request for a "bounty." What I wanted to make clear here was that while we are happy to arrange to pay developers, including security researchers, that must be arranged in advance, and not presented as a request for payment "or else."